src/studio/src/designer/backend/Factories/ModelFactory/JsonSchemaToInstanceModelGenerator.cs

```diff
@@ -324,12 +345,12 @@ private void TraverseModell(string parentPath, string parentTypeName, string pro
 
             if (path.EndsWith(".value"))
             {
-                result.Add("Texts", ExtractTexts(parentElement.Split(".").Last(), parentType));
+                result.Add("Texts", ExtractTexts(parentElement.Split(".").Last(), parentJsonSchema));
                 result.Add("IsTagContent", true);
             }
             else
             {
-                result.Add("Texts", ExtractTexts(propertyName, propertyType));
+                result.Add("Texts", ExtractTexts(propertyName, currentJsonSchema));
             }
 
             result.Add("CustomProperties", new JsonObject()); // ??
```

Comment:
Don't think the customProperties is ever used?

Reply:
I think the idea here was to allow developers to extend the metadata with some custom stuff, but not sure what exactly it was meant for. @TheTechArch any ideas?
Since this is not new code, and we probably will be skipping this whole step of generating the model metadata in the future, I suggest we leave it as is for now.

Reply:
Yep. Except from a few variable renaming operations I haven't touched the existing code to reduce the risk surface.